build_routereflector/image/cleanup.sh

```diff
@@ -5,8 +5,8 @@ set -x
 # Remove extra packages using dpkg rather than apt-get - this prevents us from
 # deleting dependent packages that we still require.
 # - Remove any temporary packages installed in the install.sh script.
-echo "Removing extra packages"
-cat /tmp/add-apt.txt | xargs xargs dpkg -r --force-depends
+#echo "Removing extra packages"
```

Comment:
Leaving commented so we can easily re-instate it later.  At the moment we can't remove because it removes a bunch of libraries that curl depends on.